### PR TITLE
WhoService client API

### PR DESCRIPTION
**Pull request by patniemeyer**
Client API for putting the device token and location.

### CODE REVIEW

File: client/flutter/lib/api/who_service.dart [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+class WhoService {
+  static final String serviceUrlStaging =
+      'https://staging.whocoronavirus.org/WhoService';
+  static final String serviceUrlProd = 'https://whocoronavirus.org/WhoService';
+  static final String serviceUrl = serviceUrlProd;
+
+  /// Put device token.
+  static Future<bool> putDeviceToken(String token) async {
+    Map<String, String> headers = await _getHeaders();
+    var postBody = '{"token": "$token"}';
```
Comment (advayDev1) [inline client/flutter/lib/api/who_service.dart]: safety: should use https://pub.dev/packages/convert to create json.

File: client/flutter/lib/api/who_service.dart [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    if (response.statusCode != 200) {
+      throw Exception("Error status code: ${response.statusCode}");
+    }
+    return true;
+  }
+
+  /// Put location
+  static Future<bool> putLocation({double latitude, double longitude}) async {
+    Map<String, String> headers = await _getHeaders();
+    var postBody = '{"latitude": "$latitude", "longitude": "$longitude"}';
```
Comment (advayDev1) [inline client/flutter/lib/api/who_service.dart]: safety: should use https://pub.dev/packages/convert to create json.

File: client/flutter/lib/api/who_service.dart (hunk 1 above)
Comment (patniemeyer) [reply client/flutter/lib/api/who_service.dart]: Fixed.

File: client/flutter/lib/api/who_service.dart (hunk 2 above)
Comment (patniemeyer) [reply client/flutter/lib/api/who_service.dart]: Fixed

**Pull request merged**